### PR TITLE
added POM to tables and registration form

### PR DESCRIPTION
**Pull request by manatskaya**
Hi Luibov,
Please review when you have a chance.
Thank you
Svitlana

### CODE REVIEW

File: cypress/e2e/app_tests/forms/headerVerificationRegistration.cy.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import { verifyAgreeToTerms, verifyConfirmPasswordInputTitle, verifyEmailInputTitle, verifyHeaderTitle, verifyNameInputTitle, verifyPasswordInputTitle, verifyTermsAndConditionsText } from "../../../models/formsModel/headerVerificationRegistrationModel";
+
+describe('Verify header and titles', () => {
+    beforeEach(() => {
+        cy.visit('/auth/register');
```
Comment (landriiuk) [inline cypress/e2e/app_tests/forms/headerVerificationRegistration.cy.js]: you can create file in models, smth like helpers.js or common.js and put function there, navigateToUrl with parameter url, and reuse this func

File: cypress/e2e/app_tests/forms/horizontalForm.cy.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import { clearEmailField, typeEmailInEmailField, clearPasswordField, typePasswordInPasswordField, clickRememberMeCheckbox, clickSignInButton,} from "../../../models/formsModel/horizontalFormModel";
+
+describe('Verify Form layout page', () => {
+    it('Clear and fill data in Horizontal form', () => {
+        cy.visit('/pages/forms/layouts');
```
Comment (landriiuk) [inline cypress/e2e/app_tests/forms/horizontalForm.cy.js]: same here about visit
Also you can add verification if user is redirected to the right url. 

cy.url().should("contain", endpoint)

File: cypress/e2e/app_tests/forms/registrationFormeValidation.cy.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+//             cy.get('body').click();
+//             cy.get('.form-control-group .caption.status-danger').as('error');
+//             cy.get('@error').eq(index).should('contain', errorMessage);
+//         }
+            
+//         verifyHeaderContent('#input-name', 0, ' Full name should contains from 4 to 50 characters ');
+//         verifyHeaderContent('#input-email', 1, ' Email should be the real one! ');
+//         verifyHeaderContent('#input-password', 2, ' Password should contain from 4 to 50 characters ');
+
+//     });
```
Comment (landriiuk) [inline cypress/e2e/app_tests/forms/registrationFormeValidation.cy.js]: Do you need help with it?

File: cypress/models/formsModel/horizontalFormModel.js [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+function clearEmailField() {
+    formPage.getEmailInHorizontalFrom().clear();
+}
+
+function typeEmailInEmailField(data) {
+    formPage.getEmailInHorizontalFrom().type(data);
+
+}
+export {clearEmailField, typeEmailInEmailField};
```
Comment (landriiuk) [inline cypress/models/formsModel/horizontalFormModel.js]: Pls put export at the bottom

File: cypress/models/formsModel/horizontalFormModel.js [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+function clearPasswordField() {
+    formPage.getPasswordInHorizontalForm().clear();
+}
+
+function typePasswordInPasswordField(value) {
+    formPage.getPasswordInHorizontalForm().type(value);
+
+}
+
+export {clearPasswordField , typePasswordInPasswordField};
```
Comment (landriiuk) [inline cypress/models/formsModel/horizontalFormModel.js]: use one export for all elements

File: cypress/models/formsModel/horizontalFormModel.js [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+
+
+
+
+
+ //cy.get('.form-horizontal [type="email"]').clear().type('test@email.com')
+       // cy.get('.form-horizontal [type="password"]').clear().type('123456)')
+        
+       // cy.get('.checkbox .label .custom-checkbox').click()
+       // cy.get('.form-horizontal').contains('Sign in').eq(0).click()
```
Comment (landriiuk) [inline cypress/models/formsModel/horizontalFormModel.js]: Pls delete comments, or if you need any help just add comment i need help here

**Review by landriiuk — APPROVED** [review]
LGTM.
If you need any help just reach me. 
Overall your task looks good!

File: cypress/models/formsModel/horizontalFormModel.js (hunk 6 above)
Comment (manatskaya) [reply cypress/models/formsModel/horizontalFormModel.js]: Deleted, thank you

**Comment by manatskaya** [comment]
Hi Luibov,
Updated according comments provided.
Thank you
Svitlana

**Pull request merged**